packages/contractkit/src/wallets/rpc-wallet.ts

```diff
@@ -51,7 +51,7 @@ export class RpcWallet extends RemoteWallet {
   }
 
   async unlockAccount(address: string, passphrase: string, duration: number) {
-    const signer = this.getSigner(address) as RpcSigner
+    const signer = this.getSigner(address)
```

Comment:
there are few more `as RpcSigner` thatyou can remove in `isAccountUnlocked` and `signTransaction`